Future Plan.md

```diff
@@ -0,0 +1,61 @@
+# Chakracore Project Future
+
+## Future Aim
+
+Future releases of Chakracore are planned by a community team of volunteers (please see history section at the end of this document for how this has come about). We will aim to:
+
+1. Prioritise ease of use for potential embedders of chakracore - these are intended as our primary users. We will aim to reflect this priority in any design decisions and by committing to never breaking the API and providing detailed and user friendly documentation.
+
+1. Achieve full standards compliance - we recognise that Chakracore is significantly behind in some areas due to its slowed pace of development, it may take some time to catch up but we will aim to do it. Any deviation from standards should be considered a bug and we will aim to fix it.
+
+1. Improve performance - we will almost certainly not be able to keep up with v8 or JSC on performance due to the smaller size of our team and limited time we have to work on this, but we will aim to continue to improve performance when we can.
```

Comment:
Would it be better to omit specific engines? That could be confusing. For example does that mean that we promise to be faster than SpiderMonkey?

Reply:
> we will aim to continue to improve performance when we can.

This possible could be enough for the performance part :)